opentech/apply/projects/forms.py

```diff
@@ -24,6 +25,35 @@ def save(self, *args, **kwargs):
         return Project.create_from_submission(submission)
 
 
+class ProjectEditForm(forms.ModelForm):
+    contact_address = AddressField()
+
+    class Meta:
+        fields = [
+            'title',
+            'contact_legal_name',
+            'contact_email',
+            'contact_address',
+            'contact_phone',
+            'value',
+            'proposed_start',
+            'proposed_end',
```

Comment:
Would it be worth some logic to confirm end > start?

Reply:
Fixed in ad8154a